.github/workflows/stale.yml

```
@@ -0,0 +1,18 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-close: 60
+          days-before-stale: -1
+          stale-issue-label: status:needs-response
+          stale-pr-label: status:needs-response
```

[`only-labels`](https://github.com/actions/stale#only-labels) is the option you're looking for.

```suggestion
          only-labels: status:needs-response
```

---

No you don't have to worry lol, I'm not keen on "update" spam.